### PR TITLE
Revert Relative Path for EditGas

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
Revert relative path for gas modal for approve transaction. Fixes[ #719](https://github.com/MetaMask/mobile-planning/issues/719)

**Screenshots/Recordings**
http://recordit.co/JwkaNlx5ze

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses https://github.com/MetaMask/mobile-planning/issues/719

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/Views/ChoosePassword/index.js [hunk 1]
```diff
@@ -69,7 +69,6 @@ import {
 import { LoginOptionsSwitch } from '../../UI/LoginOptionsSwitch';
 import generateTestId from '../../../../wdio/utils/generateTestId';
 import { scale } from 'react-native-size-matters';
-import SecureKeychain from '../../../core/SecureKeychain';
```
Comment (blackdevelopa) [inline app/components/Views/ChoosePassword/index.js:72]: `SecureKeychain` is unused in this file. Flagged by eslint.

**Review by jpuri — APPROVED** [review]

**Review by Cal-L — APPROVED** [review]
LGTM

**Comment by seaona** [comment]
QA'd the changes and looks that the issue is fixed.

- Edit Gas fees for external dapp transactions is now working fine
- Edit Gas fees for internal transactions is working fine

https://user-images.githubusercontent.com/54408225/225730277-50d4a194-4db5-484f-82c3-e525f13f14c9.mp4

File: app/components/Views/ApproveView/Approve/index.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ import { getGasLimit } from '../../../../util/custom-gas';
 import NotificationManager from '../../../../core/NotificationManager';
 import Logger from '../../../../util/Logger';
 import { trackEvent, trackLegacyEvent } from '../../../../util/analyticsV2';
-import EditGasFee1559 from '../../../UI/EditGasFee1559';
+import EditGasFee1559 from '../../../UI/EditGasFee1559Update';
```
[the remark this answers is not in the document]
Comment (blackdevelopa) [reply app/components/Views/ApproveView/Approve/index.js:24]: Ah yeah, that's where the change happened. I just checked and it seems we will need to also update the EditGasFeeLegacy path as well

[the remark this answers is not in the document]
Comment (blackdevelopa) [reply app/components/Views/ApproveView/Approve/index.js:24]: Updated. [ebaf499](https://github.com/MetaMask/metamask-mobile/pull/5973/commits/ebaf499a90b944ce4a7f5a144ceb325c2624effe)

**Review by Cal-L — APPROVED** [review]
Good catch. changes LGTM. Looks like we have an audit issue though. Let's find a way to address that

Edit: I've rebased and this should be fixed.

**Comment by blackdevelopa** [comment]
@sethkfman has a PR for the failing audit, [here](https://github.com/MetaMask/metamask-mobile/pull/5978). We can cherrypick after merge to main.

**Pull request merged**